xbmc/utils/EGLUtils.cpp

```diff
@@ -272,3 +272,81 @@ void CEGLContextUtils::SwapBuffers()
 
   eglSwapBuffers(m_eglDisplay, m_eglSurface);
 }
+
+/* --- CEGLImage -------------------------------------------*/
```

Comment:
Not that important, but why do you use `/* */` for single-line comments?

Reply:
I just find it uniform, it it bothers you I can just use `//` :smile_cat:

Reply:
Heh I don't particularly care and we don't have a guideline/rule here (like for most things), it's just uncommon in C++ afaik. Personally I've seen this mostly in pre-C99 code since `//` didn't exist then :-)